wire/wire.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package basictracer_go.wire;
```

Comment:
Thinking out loud: since the binary (and text) protocol should be shared across *all* "basictracer" impls (not just Go), eventually this .proto file should be in some sort of `basictracer-common` repo. That can/should wait until things have stabilized more, though.